src/layout/Navbar.jsx

```diff
@@ -1,21 +1,21 @@
-import { NavLink } from "react-router-dom"
-import "./Navbar.css"
+import { NavLink } from "react-router-dom";
+import "./Navbar.css";
 
 export default function Navbar() {
-    return (
-        <nav className="navbar">
-            <NavLink>Fullstack University</NavLink>
-            <menu>
-                <li>
-                    <NavLink to="/departments">Departments</NavLink>
-                </li>
-                <li>
-                    <NavLink to="/professors">Faculty</NavLink>
-                </li>
-                <li>
-                    <NavLink to="/login">Login</NavLink>
-                </li>
-            </menu>
-        </nav>
-    )
-}
\ No newline at end of file
+  return (
+    <nav className="navbar">
+      <NavLink to="/home">Fullstack University</NavLink>
```

Comment:
Previously, I set the route to link our homepage to "/" - do we want to make a new route for a "/home" URL?